test/modules/helpers-get-urls-test/fetch-file-test.js

```diff
@@ -0,0 +1,33 @@
+const test = require('ava')
+const fetchFile = require('../../../lib/helpers-get-urls/fetch-file')
+const { arraysAreEqual } = require('../../../lib/test-helpers')
+const expectedOutput = ['https://alheimsins.net/',
+  'https://alheimsins.net/projects/',
+  'https://alheimsins.net/changes/']
+
+test('existing file can be fetched', async (t) => {
+  var urlsFromJson = await (fetchFile('https://raw.githubusercontent.com/p1ho/site-validator-cli/master/test/data/urls.json'))
+  t.is(true, arraysAreEqual(urlsFromJson, expectedOutput))
```

Comment:
you can use ```t.deepEqual(urlsFromJson, expectedOutput)``` instead of arraysAreEqual.

See https://github.com/avajs/ava/blob/master/docs/03-assertions.md#built-in-assertions for all built in options

Reply:
I see, yes I was struggling a little bit with that!
Please check the newest commit.